testing/stable_sort_by_key_large.cu

```diff
@@ -69,8 +69,8 @@ void _TestStableSortByKeyWithLargeValues(void)
 
     for(size_t i = 0; i < n; i++)
     {
-        h_keys[i] = rand();
-        h_vals[i] = FixedVector<T,N>(i);
+        h_keys[i] = static_cast<unsigned int>(rand());
```

Comment:
There is a random number facility in `unittest` that we should be using here instead.

Reply:
Done.